### PR TITLE
Add post-mortem for dev env going down

### PR DESCRIPTION
**Pull request by yngvark**
## Description

Post mortem of development environment going down for team Booking.

## Motivation and Context

[KM502](https://trello.com/c/bLjI923B/502-write-post-mortem-for-env-going-down)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.
- [ ] I have updated the release notes (for the next release).

### CODE REVIEW

File: docs/post-mortems/0002-terminating-nodes.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+* We have decided to put all messages about upgrades, patches and other important messages on okctl.io. We'll create an RSS feed
+for posts, and forward those to Slack on a separate channel (TODO update with decision). With these steps, people
+have multiple options for getting information about their infrastructure.
+
+## Supporting information and documentation
+
+* For new applications made with `okctl apply application`, resource quotas are set by default. However, in theory, if somebody
+spins up lots of pods wihout resource quotas, this could happen again.
+* A final thought: This is what can happen when we're establishing a culture of "you own it, you run it", where people run their
```
Comment (deifyed) [inline docs/post-mortems/0002-terminating-nodes.md:91]: 🥰

**Review by deifyed — APPROVED** [review]
Nothing to add. Well written

File: docs/post-mortems/0002-terminating-nodes.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+
+  * In AWS console, we set Auto Scaling Groups min and max capacity from 1 to 0, and then back again. At the time, we had an
+  idea it would fix issues with the node. In hindsight, this was probably not necessary.
+
+  * We run the resource quota patch to set resource quotas on most pods except Loki.
+
+  * We create a backup of Loki logs in development, and test that it's possible to restore. We then apply the persistent
+    storage patch for Loki, and restore the logs. We then run the resource quota patch again, this time enabling it for Loki also.
+
+  * We then run the resource quota patch, without Loki, in production. As it's getting late, we delay the rest of the fix to
```
Comment (fredriv) [inline docs/post-mortems/0002-terminating-nodes.md:31]: Do you have any timestamps for the other actions? Or an end time for the support, so we can see how much time was spent on the incident?

**Review by fredriv — APPROVED** [review]
Looks good! :+1:

If you have more timestamps or alternatively duration of support/incident that would be nice to see as well.

File: docs/post-mortems/0002-terminating-nodes.md (hunk 2 above)
Comment (yngvark) [reply docs/post-mortems/0002-terminating-nodes.md]: Done!

**Pull request merged**